examples/custom-resources/transport-server-sni/tcp-echo-server.yaml

```diff
@@ -0,0 +1,39 @@
+ apiVersion: apps/v1
+ kind: Deployment
+ metadata:
+   name: tcp-echo-server
+ spec:
+   replicas: 1
+   selector:
+     matchLabels:
+       app: tcp-echo-server
+   template:
+     metadata:
+       labels:
+         app: tcp-echo-server
+     spec:
+       containers:
+       - name: tcp-echo-server
+         image: alpine
+         command: ["/bin/sh"]
+         args:
+         - -c
+         - |
+           apk add --no-cache socat
+           echo "TCP Echo Server starting..."
+           socat -v tcp-listen:7000,fork,reuseaddr,keepalive exec:'/bin/cat'
```

Comment:
do we need `socat` here?  Alpine comes with netcat (`nc`) already

Reply:
No. Thanks this is better will update.